.env.example

```
@@ -62,10 +62,12 @@ NUXT_OPENAI_API_KEY=your-openai-api-key-here
 # For testing
 NUXT_TEST_DATABASE_URL=postgres://postgres:@localhost:5432/getquillio_test
 NUXT_TEST_APP_URL=http://localhost:3000 # App URL for e2e test
-NUXT_TEST_EMAIL=test@getquillio.com
+NUXT_TEST_EMAIL="test@getquillio.com" # Must exist in the test DB and be verified
+NUXT_TEST_ORGANIZATION_ID="" # Optional: force active org for e2e tests
 NUXT_TEST_PASSWORD=test-password
 NUXT_ADMIN_EMAIL=admin@getquillio.com # Admin user email for e2e tests
 NUXT_ADMIN_PASSWORD=admin-password # Admin user password for e2e tests
+TARGET_EMAIL=bamboo.chow@gmail.com # Required by scripts/extract-bamboo-examples.ts
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)